i18n/en-US.properties

```diff
@@ -1002,6 +1002,64 @@ boxui.searchForm.searchLabel = Search query
+boxui.securityControls.desktopDownloadOwnersEditors = Download restricted on Box Drive, except Owners/Co-Owners/Editors.
+# Bullet point that summarizes desktop download restrictions applied to classification, when restriction applies to external users. Box Drive is a product name and not translated
+boxui.securityControls.downloadExternal = Download restricted on Box Drive for external users.
+# Bullet point that summarizes external collaboration blocked restriction applied to classification
+boxui.securityControls.externalCollabBlock = External collaboration restricted.
+# Bullet point that summarizes external collaboration restriction applied to classification
+boxui.securityControls.externalCollabDomainList = External collaboration limited to approved domains.
+# Bullet point that summarizes mobile download restrictions applied to classification, when restriction applies to external users
+boxui.securityControls.mobileDownloadExternal = Download restricted on mobile for external users.
+# Bullet point that summarizes mobile download restrictions applied to classification, when restriction applies to external users and managed users except Owners/Co-Owners
+boxui.securityControls.mobileDownloadExternalOwners = Download restricted on mobile for external users, except Owners/Co-Owners.
+# Bullet point that summarizes mobile download restrictions applied to classification, when restriction applies to external users and managed users except Owners/Co-Owners/Editors
+boxui.securityControls.mobileDownloadExternalOwnersEditors = Download restricted on mobile for external users, except Owners/Co-Owners/Editors.
+# Bullet point that summarizes mobile download restrictions applied to classification, when restriction applies to managed users except Owners/Co-Owners
+boxui.securityControls.mobileDownloadOwners = Download restricted on mobile, except Owners/Co-Owners.
+# Bullet point that summarizes mobile download restrictions applied to classification, when restriction applies to managed users except Owners/Co-Owners/Editors
+boxui.securityControls.mobileDownloadOwnersEditors = Download restricted on mobile, except Owners/Co-Owners/Editors.
+# Bullet point that summarizes collaborators shared link restriction applied to classification
+boxui.securityControls.sharingCollabAndCompanyOnly = Shared links allowed for collaborators only.
+# Bullet point that summarizes shared link restriction applied to classification
+boxui.securityControls.sharingCollabOnly = Shared links cannot be made publicly accessible.
+# Short summary displayed for classification when sharing, download and app download restrictions are applied to it
+boxui.securityControls.shortAllRestrictions = Sharing, download and app and restrictions apply
+# Short summary displayed for classification when an application download restriction is applied to it
+boxui.securityControls.shortApp = Application restrictions apply
```

Comment:
application --> app to match other places?

Reply:
Same here, will check with product/design